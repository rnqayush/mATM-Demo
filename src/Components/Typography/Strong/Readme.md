Text With `Strong` Component

```jsx
import Text from '../Text';
<Text>
  Lorem Ipsum is <Strong>simply dummy</Strong> text of the printing and typesetting industry.
</Text>;
```
