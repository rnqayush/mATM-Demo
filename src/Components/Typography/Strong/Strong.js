import React from 'react';
import PropTypes from 'prop-types';

import WrapperText from './Strong.style';

const Strong = ({children, testID}) => <WrapperText testID={testID}>{children}</WrapperText>;

Strong.propTypes = {
  children: PropTypes.node.isRequired,
  testID: PropTypes.string,
};

Strong.defaultProps = {
  testID: undefined,
};
/** @component */
export default Strong;
