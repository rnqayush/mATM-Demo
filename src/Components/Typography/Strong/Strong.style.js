import styled from 'styled-components/native';

const WrapperText = styled.Text`
  font-family: ${({theme}) => theme?.FontFamily?.BOLD};
`;

export default WrapperText;
