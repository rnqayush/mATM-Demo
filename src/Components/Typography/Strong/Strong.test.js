import React from 'react';
import {render} from '../../../../test/test-utils';

import Strong from './index';

describe('Strong', () => {
  it('Strong renders correctly', () => {
    expect(render(<Strong testID="boldText">SomeText</Strong>)).toMatchSnapshot();
  });
});
