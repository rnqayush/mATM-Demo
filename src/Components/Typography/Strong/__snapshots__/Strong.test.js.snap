// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Strong Strong renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "fontFamily": "Roboto-Bold",
      },
    ]
  }
  testID="boldText"
>
  SomeText
</Text>
`;
